pages/secure/sec_url_whitelist.rst

```diff
@@ -0,0 +1,42 @@
+.. _sec_url_whitelist:
+
+*******************************
+The URL whitelist
```

Comment:
I think all instances of "Whitelist" should be capitalized in this doc, since it's the name of the feature.
```suggestion
The URL Whitelist
```